### ISSUE TITLE
Review ligatures in small caps

### ISSUE THREAD

**Issue by gorjious** [googlefonts/googlesans#69]
Ligatures remain unchanged when converting from lowercase to small caps (smcp feature). 

<img width="1302" alt="Screenshot 2020-07-07 at 09 53 50" src="https://user-images.githubusercontent.com/23149420/86808720-59789e00-c038-11ea-87c9-49c5fb7067e9.png">

**Comment by chrissimpkins** [comment]
Thanks Gor!

**Comment by chrissimpkins** [comment]
It looks like we need to add ligature decomposition in the smcp feature to address this:

```fea
feature smcp {
# snip...
    sub f_f_b by f.sc f.sc b.sc;
    sub f_f_h by f.sc f.sc h.sc;
    sub f_f_i by f.sc f.sc i.sc;
    sub f_f_j by f.sc f.sc j.sc;
    sub f_f_k by f.sc f.sc k.sc;
    sub f_f_l by f.sc f.sc l.sc;
    sub f_f_t by f.sc f.sc t.sc;
    sub r_f_f by r.sc f.sc f.sc;
    sub f_b by f.sc b.sc;
    sub f_f by f.sc f.sc;
    sub f_h by f.sc h.sc;
    sub f_i by f.sc i.sc;
    sub f_j by f.sc j.sc;
    sub f_k by f.sc k.sc;
    sub f_l by f.sc l.sc;
    sub f_t by f.sc t.sc;
    sub r_f by r.sc f.sc;
    sub r_t by r.sc t.sc;
    sub t_f by t.sc f.sc;
    sub t_t by t.sc t.sc;
}
```

**Comment by chrissimpkins** [comment]
We also need shaping regression tests with these ligature strings + smcp feature on

**Comment by chrissimpkins** [comment]
Fixed in https://github.com/googlefonts/googlesans/commit/3d11a2679715657b6d2e4f6d5f9bb57aa598e66d

Will be released in v4.000

**Comment by chrissimpkins** [comment]
Fix confirmed.  Thanks so much Gor!  Good eye

<img width="1855" alt="2021-03-05_08-48-28" src="https://user-images.githubusercontent.com/4249591/110124095-b0816f00-7d8f-11eb-8770-41bc1a165184.png">
